### PR TITLE
Changed sourcemaps to sourceMaps

### PR DESCRIPTION
**Pull request by wagerfield**
Fixes https://github.com/fuse-box/fuse-box/issues/355

### CODE REVIEW

**Comment by wagerfield** [comment]
**NOTE:** I haven't updated any docs, tests or code for `SourceMapPlainJsPlugin`. I wasn't entirely sure what to do here, so if this needs updating also—please guide me.

**Comment by nchanged** [comment]
Thanks! We need to keep `sourcemaps` as well, In order for people to migrate. You think you can keep it there?

**Comment by wagerfield** [comment]
Updated `sourceMaps` logic to support `sourcemaps` too as requested.

**Comment by wagerfield** [comment]
Not related to the title of this PR, but there is a fix for the `ReplacePlugin` too. This small update will replace all matches as opposed to just the first occurrence.

File: src/core/FuseBox.ts [hunk 1]
```diff
@@ -177,18 +178,19 @@ export class FuseBox {
             //this.context.sourceMapConfig = opts.sourceMap;
         }
 
-        if (opts.sourcemaps) {
+        const sourceMaps = opts.sourceMaps || opts.sourcemaps
```
Comment (nchanged) [inline src/core/FuseBox.ts]: Great!

File: src/plugins/ReplacePlugin.ts [hunk 2]
```diff
@@ -9,7 +9,8 @@ export class ReplacePluginClass implements Plugin {
     transform(file: File) {
         for (let key in this.options) {
             if (this.options.hasOwnProperty(key)) {
```
Comment (wagerfield) [inline src/plugins/ReplacePlugin.ts]: Is this condition needed?

Comment (nchanged) [inline src/plugins/ReplacePlugin.ts]: Typescript ... safe properties =)

**Pull request merged**